lizmap/modules/lizmap/controllers/edition.classic.php

```diff
@@ -797,6 +806,10 @@ public function saveFormDataToDb($form){
     foreach($fields as $ref){
       // Get and filter the posted data foreach form control
       $value = $form->getData($ref);
+      
+      if(is_array($value)){
+        $value = '{'.join($value,',').'}';
```

Comment:
It seems that join is not a PHP method. Can you correct this line with implode method.